examples/quickstart/tutorial/hadoop/docker/Dockerfile

```diff
@@ -19,6 +19,11 @@ FROM centos:7
 
 USER root
 
+# CentOS is EOL, have to use vault
```

Comment:
@findingrish , just for clarification, is CentOS 7 EOL or is it `mirror.centos.org` that is not up anymore?

Reply:
[CentOS 7](https://www.redhat.com/en/topics/linux/centos-linux-eol#:~:text=CentOS%20Linux%207%20reached%20end%20of%20life%20(EOL)%20on%20June,can%20help%20ease%20your%20migration) is EOL.

Reply:
And to be more specific, CentOS as an OS is EOL.